olp-cpp-sdk-dataservice-read/src/CatalogClientImpl.cpp

```diff
@@ -249,7 +249,7 @@ client::CancellationToken CatalogClientImpl::PrefetchTiles(
     callback(response);
   };
 
-  auto token = prefetch_provider_->PrefetchTiles(request, callback);
+  auto token = prefetch_provider_->PrefetchTiles(request, request_callback);
```

Comment:
Typo?

Reply:
Can you explain? In case we will pass only `callback` there, the pending requests will contain 'dead' request keys of already processed PrefetchTiles requests. Thus, we need to pass the `request_callback` instead.

Reply:
Get it now